client/checkout/blocks/confirm-upe-payment.js

```diff
@@ -55,15 +57,17 @@ export default async function confirmUPEPayment(
 		// Afterpay requires shipping details to be passed. Not needed by other payment methods.
 		if ( PAYMENT_METHOD_IDS.AFTERPAY_CLEARPAY === selectedUPEPaymentType ) {
 			confirmParams.shipping = {
-				name,
-				phone: billingData.phone || '-',
+				name:
+					`${ shippingData.first_name } ${ shippingData.last_name }`.trim() ||
+					'-',
+				phone: shippingData.phone || '-',
```

Comment:
We skipped the phone in another PR. Here we keep it. Please pay more attention to consistency - it's important for a product that we'll maintain in the long run.

Reply:
In the classic checkout there is no phone field for Shipping data. However in the block checkout, there is a separate phone field for Shipping too. Coming to think of it though, since Afterpay does not necessarily need it ( per their sample ), I think I will skip it.

Reply:
I have removed it here - 519ce3e. Thanks.